systemservices/marketplace/src/events/serviceVersionCreated.ts

```diff
@@ -1,12 +1,13 @@
 import Service, { EmitEventReply } from "mesg-js/lib/service/service"
 import { EventLog } from "web3/types";
 import { hexToAscii } from "../contracts/utils";
+import { getManifest } from "../contracts/manifest";
 
-export = (mesg: Service, event: EventLog): Promise<EmitEventReply | Error> => {
+export = async (mesg: Service, event: EventLog): Promise<EmitEventReply | Error> => {
+  const manifest = await getManifest(hexToAscii(event.returnValues.manifestProtocol), hexToAscii(event.returnValues.manifest))
```

Comment:
```suggestion
```

I think event should not return the content of manifest. I see event as small and fast message to trigger other function. In this case, the triggered function can get the manifest if it needs to.
Otherwise, why not returning also ALL the service's data?

Reply:
Are you saying that this should be moved to the method that calls this function or that it should simply disappear and we just return the raw data without interpretation of these data.

I would actually be ok to return all the service data and have the same structure than the `getService` or `listServices` that would make even more sense.